lang/po/Firmware_cs.po

```diff
@@ -206,7 +206,7 @@ msgstr "Podsviceni"
 #. MSG_TITLE_COMMUNICATION_ERROR c=20
 #: ../../Firmware/mmu2/errors_list.h:147 ../../Firmware/mmu2/errors_list.h:185
 msgid "COMMUNICATION ERROR"
-msgstr ""
+msgstr "CHYBA KOMUNIKACE"
```

Comment:
do we really want the MMU messages to be present in 3.12?

Reply:
This PR is for MK3 including the MMU2 messages. As mentioned above a cherry-pick to MK3_3.12 will be done after merge.